ci-automation/ci_automation_common.sh

```diff
@@ -507,3 +507,17 @@ function apply_local_patches() {
   done
 }
 # --
+
+# Returns 0 if passed value is either "1" or "true" or "t", otherwise
+# returns 1.
+function bool_is_true() {
+    case "${1}" in
+        1|true|t)
```

Comment:
```suggestion
        1|true|t|yes|y)
```
If thee values are allowed I might get confused which and also try `y` ;D

Reply:
That's a good idea, thanks.